var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyMdanalysis(PythonPackage):
+    """MDAnalysis is a Python toolkit to analyze molecular dynamics
+    trajectories generated by a wide range of popular simulation
+    packages including DL_Poly, CHARMM, Amber, NAMD, LAMMPS, and
+    Gromacs. (See the lists of supported trajectory formats and
+    topology formats.)"""
+
+    homepage = "http://www.mdanalysis.org"
+    url      = "https://pypi.python.org/packages/9f/56/a4cae14e45aa893a3578905a97e17a325d663e424736366adbc134e72dce/MDAnalysis-0.15.0.tar.gz#md5=19e5a8e6c2bfe85f6209d1d7a36e4f20"
```

Comment:
Can you change this to:
https://pypi.io/packages/source/M/MDAnalysis/MDAnalysis-0.15.0.tar.gz
This will allow us to download different versions from the same URL.